src/main/java/com/shaft/dsl/webElements/Element.java

```diff
@@ -0,0 +1,33 @@
+package com.shaft.dsl.webElements;
+
+import com.shaft.gui.element.ElementActions;
+import com.shaft.validation.Validations;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Element {
+    By locator;
+    WebDriver driver;
+    ElementActions elementActions;
```

Comment:
Can we privatize the variables?

Reply:
by default it is private 
because it is the pillar of OOP

Reply:
I think by default it is protected... Are you sure?

Reply:
https://stackoverflow.com/questions/16164902/what-is-the-default-access-modifier-in-java#:~:text=The%20Default%20access%20modifier%20is,only%20from%20the%20same%20package.&text=Your%20constructor's%20access%20modifier%20would,but%20the%20constructor%20will%20not.

Reply:
package-private is closer to protected than it is to private (class private I mean)
Do you think it's better to leave it accessible from any class in the package? Or to make it only accessible within the class?

Reply:
I need it so 
so that I can work on them in the sub classes 
i have even doughts that I will need to convert them to protected if I needed them in the subclasses

Reply:
Sounds good to me 👌